.gitignore

```diff
@@ -28,7 +28,6 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+# local environment set up
 application-local.yml
-
-# merged .graphqls files used in code generation
-ui/src/generated/schema.graphqls
\ No newline at end of file
+local.env
```

Comment:
Question: do we want to add any info about this to a README?

Reply:
YES!  I was going to get that written up soon